TRIAGING.md

```diff
@@ -0,0 +1,118 @@
+## The Triaging Process at `conda`
+
+"Triaging" refers to the process of assessing the priority of incoming issues. Below is a high-level diagram of the flow of tickets:
+
+```mermaid
+flowchart LR
+    issue_new(New Issues)
+
+    subgraph board_triaging [Triaging Board]
+    col_triaging{{Triaging}}
+    col_support{{Support}}
+    col_ready{{Ready}}
+    col_closed{{Closed}}
+    col_triaging<-->col_support-->col_closed
+    col_triaging-->col_ready
+    col_triaging-->col_closed
+    end
+
+    issue_new-->col_triaging
+   col_ready-->Backlog
+```
+
+In order to explain how various `conda` issues are evaluated, the following document will provide information about our triaging process in the form of an FAQ.
+
+
+### Why triage?
+
+At the most basic "bird's eye view" level, triaged issues will fall into the category of four main priority levels:
+
+- Do now
+- Do sometime
+- Provide user support
+- Never do (_i.e._, close)
+
+At its core, triaging enables new issues to be sorted into these four categories, which helps to ensure that they will be processed at a velocity similar to or exceeding the rate at which new issues are coming in. One of the benefits of actively triaging issues is to avoid engineer burnout and make necessary work sustainable; this is done by eliminating a never-ending backlog that has not been reviewed by any maintainers.
+
+There will always be broad-scope design and architecture implementations that the `conda` team will be interested in pursuing; by actively triaging issues, the triaging engineers will be able to more easily track and tackle both specific and big-picture goals.
+
+### Who triages?
+
+<!-- This section is still a WIP, there is no official triaging engineer team yet to link to -->
+
+Triaging engineers are a `conda` governance [sub-team](https://github.com/conda-incubator/governance#sub-teams); they are a group of Anaconda and community members who are responsible for making decisions regarding closing issues and setting feature work priorities, amongst other triage-related tasks. The list of current members is listed [here](future link)
```

Comment:
We *can* request a new sub-team and add ourselves to it. Not sure when the right time for this is, but it would signal that we're open for contributions from people in the community.